build_to_jdi-light.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -v
+
+# VARIABLES
+# Feel free to edit if your directory names or branches differ
+TEST_SITE_DIR=$(pwd)
+JDI_LIGHT_DIR="jdi-light"
+JDI_LIGHT_BRANCH="gh-pages"
+SERVER_PORT=8001
+
+# Help
+if [[ ($1 == "-h") || ("$1" == "--help") ]]
+then
+  echo "
+  This bash script displays the angular-site in browser for visual verification
+  If you are satisfied with result, the job proceeds and places the built site into '${JDI_LIGHT_DIR}/angular' folder
+  The server is then started at localhost:${SERVER_PORT}
+
+  Prerequisites (please read carefully):
+     - Node.JS and npm are installed
+     - Angular CLI is installed(npm install -g @angular/cli)
+     - Python3 is installed
+     - This sh script is executed in bash (zsh, git-bash and other shells should work)
+     - This script is executed from test-site directory (${TEST_SITE_DIR})
+     - jdi-light project is checked out into '../${JDI_LIGHT_DIR}', active branch is ${JDI_LIGHT_BRANCH}
+     - One can modify some variables of this script if needed, see VARIABLES section
+  "
+  exit 0
+fi
+
+# Start
+# 0. Check the active branch of jdi-light
+BRANCH=$(cd ../${JDI_LIGHT_DIR} && git symbolic-ref --short HEAD)
+if [[ ${BRANCH} != ${JDI_LIGHT_BRANCH} ]]
+then
+  echo "Please change active branch of jdi-light to ${JDI_LIGHT_BRANCH}."
+  exit 1
+fi
+
+# 1. Move to ./angular-site directory and perform npm install if needed
+[ ! -d "angular-site" ] && echo "angular-site dir is not found" && exit 1
+cd angular-site
+
+[ ! -d "node_modules" ] && echo "node_modules dir is not found - performing npm install" && npm install
+
+# 2. Verify the result visually (output is muted in order not to cover the following echo output)
+ng serve --open > /dev/null &
+echo "Test site is going to be built and displayed in your browser in approximately 20 seconds"
+echo "Press Y if you are satisfied with the result and would like to proceed with building the js files and placing them to ${JDI_LIGHT_DIR}."
+echo "This action cannot be undone."
+
+# 3. Build and place the files in required folder
+#reads the y/n
+read -n 1 -r keypressed
```

Comment:
why do we need to ask it? we could not want to build or what? I don't understand ((

Reply:
This prompt allows user to interrupt the script if he is not satisfied with performed changes.
I've added additional info to help section that describes the script logic.